client/components/forms/form-select/style.scss

```diff
@@ -48,6 +48,9 @@
 		background:
 			url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAxOCAxOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTE1IDYuNzUwMDFMOSAxMi43NUwzIDYuNzUwMDFMNC4wNjA1IDUuNjg5NTFMOSAxMC42MjlMMTMuOTM5NSA1LjY4OTUxTDE1IDYuNzUwMDFaIiBmaWxsPSIjQzNDNEM3Ii8+Cjwvc3ZnPgo=)
 			no-repeat right 10px center;
+		background-color: var(--studio-gray-0);
+		border-color: var(--gray-gray-10, #c3c4c7);
```

Comment:
Any reason for the color fallback here?

Reply:
Mentioned in the description:

> I am a bit confused about what css variables should be present and what theme object represent the latest styles. For example, the fields use a border-color of var(--gray-gray-10, #c3c4c7); across the app, but locally --gray-gray-10 is not defined. 

I found multiple references of this: https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso+var%28--gray-gray-10%2C+%23c3c4c7%29&type=code

It exists in [some places](https://github.com/Automattic/wp-calypso/blob/0990db38a4f013587d55908af2796950346d9a2e/packages/composite-checkout/src/lib/swatches.ts#L22) but not others.